sql/2020/09_Performance/median_lcp_score_by_month.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
+# Median of Largest contentful paint 75% score by month
+
+SELECT
+  date,
+  device,
+  median_p75_lcp
+FROM (
+  SELECT
+    date,
+    device,
+    PERCENTILE_CONT(p75_lcp, 0.5) OVER(PARTITION BY date, device) AS median_p75_lcp
```

Comment:
I hadn't seen this function before! I've always used `APPROX_QUANTILES`. I wonder how it performs with millions of records.

Reply:
Yeah, it's 1.4 s vs 7.3 s.
Updated to:
`APPROX_QUANTILES(p75_lcp, 2 RESPECT NULLS)[OFFSET(1)] AS approx_median_p75_lcp`